mappings/bloblang/AWS/eks/v1.2.0/samples/ocsf_eks.json

```diff
@@ -0,0 +1,423 @@
+{
+    "activity_id": 1,
+    "activity_name": "Create",
+    "actor": {
+      "session": {
+        "credential_uid": "EXAMPLEUIDTEST",
+        "issuer": "arn:aws:iam::123456789012:role/example-test-161366663-NodeInstanceRole-abc12345678912",
+        "uid": "i-12345678901"
+      },
+      "user": {
+        "groups": [
+          {
+            "name": "system:bootstrappers"
+          },
+          {
+            "name": "system:nodes"
+          },
+          {
+            "name": "system:authenticated"
+          }
+        ],
+        "name": "system:node:ip-192-001-02-03.ec2.internal",
+        "type_id": 0,
+        "uid": "heptio-authenticator-aws:123456789012:ABCD1234567890EXAMPLE"
+      }
+    },
+    "api": {
+      "operation": "create",
+      "request": {
+        "uid": "f47c68f2-d3ac-4f96-b2f4-5d497bf79b64"
+      },
+      "response": {
+        "code": 201
+      },
+      "version": "v1"
+    },
+    "category_name": "Application Activity",
+    "category_uid": 6,
+    "class_name": "API Activity",
+    "class_uid": 6003,
+    "cloud": {
+      "account": {
+        "uid": "arn:aws:sts::123456789012:assumed-role/example-test-161366663-NodeInstanceRole-abc12345678912/i-12345678901"
+      },
+      "provider": "AWS"
+    },
+    "http_request": {
+      "url": {
+        "path": "/api/v1/nodes"
+      },
+      "user_agent": "kubelet/v1.21.2 (linux/amd64) kubernetes/729bdfc"
+    },
+    "message": "ResponseComplete",
+    "metadata": {
+      "log_level": "RequestResponse",
+      "product": {
+        "feature": {
+          "name": "Elastic Kubernetes Service"
+        },
+        "name": "Amazon EKS",
+        "vendor_name": "AWS",
+        "version": "audit.k8s.io/v1"
+      },
+      "profiles": [
+        "cloud",
+        "datetime"
+      ],
+      "version": "1.2.0"
+    },
+    "observables": [
```

Comment:
Could use HTTP User Agent observable here. As well as dynamically creating one for Resource UID where `resources[].uid` is not `null`. And a few for Hostname. Depends how crazy you want to get.

Reply:
I need to add all the new observables to all of the mapping. I will make a new PR for that.